project/Testing.scala

```diff
@@ -55,7 +59,7 @@ object Testing {
     (testOnly in Test) := ((testOnly in Test) dependsOn validMySqlHost).evaluated,
 
     parallelExecution in Test := false
-  ) ++ MinnieKenny.testSettings
+  ) ++ (if (sys.props.getOrElse("secrets.skip", false) != "true") MinnieKenny.testSettings else List())
```

Comment:
I chose this option name somewhat arbitrarily, if it doesn't make sense it can be changed. Alternatively, we could remove Minnie-Kenny being run by sbt; Orch doesn't have it, and it's run manually in the github CI.

Reply:
To confirm: Orch doesn't check for the secrets leaks commited in `minniekenny.gitconfig`? Is that a future ticket that can be linked in a comment?

Reply:
Orch doesn't have any minnie-kenny anything of any kind; I'm not aware of any discussion around it, @davidangb do you know of anything?

Reply:
I don't see any tickets in the backlog. It is used by most other services (like rawls and sam) so maybe we should add it

Reply:
for my clarification: this discussion of orch is tangential to this PR, right?

yeah, if orch does not have secrets-detection, it absolutely needs it. I'd prefer to check with appsec first on the progress of their centralized/cross-repo secrets detection before committing to any work, if appsec is already handling it let's just let them.

Reply:
I think it's tangential. The question came from the switch to skip testing for git-secrets via the line above that says `sys.props.getOrElse("secrets.skip"`. From the comments, it sounded like orch was the repo/app that would add use this toggle instead of adding secrets tests?

Reply:
@zarsky-broad please confirm, but my understanding is this toggle is specific to Rawls, and is called from the change in this PR in docker/install.sh.

Reply:
Yeah. The old build image for Rawls didn't include git at all, so the secrets check was skipped silently. The new image has git, so the secrets check was failing here because it didn't have git-secrets. I added an optional flag to skip it that's only used in the `install.sh`.

The alternative is to not have sbt run the check at all, since it's explicitly run in the Github CI workflow.

Reply:
Ah, I see the problem:

> Orch doesn't have it, and it's run manually in the github CI.

Orch has no minnie-kenny at all. Separately, the rawls github workflow runs minnie-kenny.

Reply:
I think there's a scenario here where you're comparing a Boolean to a String. My quick test in the repl shows that this can cause problems. Recommend coercing everything the same type.